cubi_isa_templates/isatab-ghga-sc_rnaseq-multiplex/cookiecutter.json

```diff
@@ -0,0 +1,161 @@
+  "data_access_committee_ega_accession": "",
+
+  "sample_names": "alpha,beta,gamma",
+  "a_measurement_type": "transcription profiling",
+  "_measurement_types": {
+      "transcription profiling" : {
+         "accession" : "http://purl.obolibrary.org/obo/OBI_0000424",
+         "source" : "OBI"
+      }
+  },
+  "a_technology_type": "nucleotide_sequencing",
+  "_technology_types": {
+      "nucleotide_sequencing" : {
+         "accession" : "http://purl.obolibrary.org/obo/OBI_0000626",
+         "source" : "OBI"
+      }
+  },
+
+  "lib_kit": [
+    "SC3Pv3",
+    "SC3Pv2",
+    "SC5P-PE",
+    "SC5P-R2",
+    "threeprime",
+    "fiveprime"
+  ],
+  "organism": [
+    "Homo sapiens",
+    "Mus musculus"
+  ],
+  "batch": "1",
+  "_lib_kits": {
+    "SC3Pv3": {
+      "name": "SC3Pv3"
+    },
+    "SC3Pv2": {
+      "name": "SC3Pv2"
+    },
+    "SC5P-PE": {
+      "name": "SC5P-PE"
+    },
+    "SC5P-R2": {
+      "name": "SC5P-R2"
+    },
+    "threeprime": {
+      "name": "threeprime"
+    },
+    "fiveprime": {
+      "name": "fiveprime"
+    }
+  },
+  "_organisms": {
+    "Homo sapiens": {
+      "name": "Homo sapiens",
+      "taxon": "9606"
+    },
+    "Mus musculus": {
+      "name": "Mus musculus",
+      "taxon": "10090"
+    }
+  },
+  "instrument": "Illumina NovaSeq 6000",
+  "center_name": "BIH Genomics Technology Platform",
+  "center_contact": "",
+  "date": "2024-11-12",
+
+  "sc_dissociation_method": [
+    "Enzymatic dissociation",
+    "Mechanical dissociation", 
+    "Cold protease-free dissociation"
+  ],
+  "sc_library_type": [
+    "GEX",
+    "GEX+antibody",
+    "GEX+VDJ",
+    "GEX+other",
+    "GEX+antibody+VDJ",
+    "GEX+other+another"
+  ],
+  "sc_multiplex_genotype": ["FALSE", "TRUE"],
+  "sc_multiplex_sample": ["FALSE", 
+    "Probe Barcode",
+    "Cell Multiplexing Oligo",
+    "Hashtag Antibody",
+    "On-chip multiplexing"], 
+  "sc_multiplex_oligo": [
+    "BC001",
+    "CMO301",
+    "OB1",
+    "TotalSeq-A"],
+  "sc_modality_1": [
+    "Gene Expression",
+    "VDJ-T",
+    "VDJ-B", 
+    "Antibody Capture",
+    "Multiplexing Capture"
+  ],
+  "sc_modality_2": [
+    "",
+    "Gene Expression",
+    "VDJ-T",
+    "VDJ-B", 
+    "Antibody Capture",
+    "Multiplexing Capture"
+  ],
+  "sc_modality_3": [
+    "",
+    "Gene Expression",
+    "VDJ-T",
+    "VDJ-B", 
+    "Antibody Capture",
+    "Multiplexing Capture"
+  ],
+
+  "study_title": "{{cookiecutter.investigation_title}}",
+  "s_file_name": "{{cookiecutter.investigation_title|replace(' ', '_')}}",
+  "assay_prefix": "{{cookiecutter.s_file_name|lower}}",
+  "a_measurement_abbreviation": "scRNA_seq",
+  "assay_name": "{{cookiecutter.a_measurement_type|replace(' ', '_')}}_{{cookiecutter.a_technology_type|replace(' ', '_')}}",
+  "sample_type": "",
+  "lib_strategy": "scRNA-seq",
+  "lib_selection": "PolyA",
+  "lib_layout": [ "PAIRED", "SINGLE" ],
+  "__output_dir": "{{cookiecutter.investigation_title}}_isatab"
```

Comment:
We usually control this variable tightly via extra_context, see here: https://github.com/bihealth/cubi-tk/blob/65df0880be65db8dab864244de9869626665a353/src/cubi_tk/isa_tpl/__init__.py#L74.

The default should therefore be sth generic, like <template_name>_isatab.